roles/fio-distributed/templates/client_store.yaml

```diff
@@ -0,0 +1,44 @@
+---
+kind: Job
+apiVersion: batch/v1
+metadata:
+  name: '{{ meta.name }}-fio-client'
+  namespace: '{{ operator_namespace }}'
+spec:
+  ttlSecondsAfterFinished: 600
+  template:
+    metadata:
+      labels:
+        app: fiod-client
+    spec:
+      containers:
+      - name: fio-client
+        image: "quay.io/benchmark-operator/fio:latest"
+        command: ["/bin/sh", "-c"]
+        args:
+          - "mkdir -p {{results.path}}/fiod-{{ uuid }};
+             cd {{results.path}}/fiod-{{ uuid }};
+             cat /tmp/fio/fiojob | tee {{results.path}}/fiod-{{ uuid }}/fiojob;
+             cat /tmp/host/hosts | tee {{results.path}}/fiod-{{ uuid }}/fiohosts;
+             fio --client=/tmp/host/hosts /tmp/fio/fiojob --output-format=json --output={{results.path}}/fiod-{{ uuid }}/fio-result.json;
+             sed -i '/{/,$!d' {{results.path}}/fiod-{{ uuid }}/fio-result.json"
+        volumeMounts:
+        - name: fio-volume
+          mountPath: "/tmp/fio"
+        - name: host-volume
+          mountPath: "/tmp/host"
+        - name: result-data
+          mountPath: "{{results.path}}"
+      volumes:
+      - name: fio-volume
+        configMap:
+          name: "fio-test"
+          defaultMode: 0777
```

Comment:
Do we really need RWX?

Reply:
I think I just copy pasted it into templates.

Reply:
is this referring to K8S accessMode or POSIX chmod value?   Looks like chmod to me.  You don't need execute mode, could just be 0666.

Reply:
Not sure it was a copy paste to template it so leaving it as is.